dictionaries/CallMap.php

```diff
@@ -13683,7 +13683,7 @@
 'ssh2_exec' => ['resource|false', 'session'=>'resource', 'command'=>'string', 'pty='=>'string', 'env='=>'array', 'width='=>'int', 'height='=>'int', 'width_height_type='=>'int'],
 'ssh2_fetch_stream' => ['resource|false', 'channel'=>'resource', 'streamid'=>'int'],
 'ssh2_fingerprint' => ['string|false', 'session'=>'resource', 'flags='=>'int'],
-'ssh2_forward_accept' => ['resource|false', 'session'=>'resource'],
+'ssh2_forward_accept' => ['resource|false', 'listener'=>'resource'],
```

Comment:
The extension defined 2 optional parameters (host and port) that are unused in the code and undocumented in PHP docs but are reflected.